Automattic/newspack-plugin
# feat(popups wizard): add archive page permalink as preview URL

### All Submissions:






### Changes proposed in this Pull Request:



Enable previewing campaign prompts on an archive page. This is needed to [preview the new inline prompts placement inside the archive pages](https://github.com/Automattic/newspack-popups/pull/621).

### How to test the changes in this Pull Request:

1. If it's not released, please checkout #plugin PR
2. Pull this PR, and add a new `inline` prompt from the campaign menu (you need the newspack-campaign plugin to be active).
3. In the `placement` option select `In archive pages`.
4. Preview the new prompt from the prompt list and observe that it loads on a category archive page.

### Other information:
